Tests/MathLib/TestLinearSolver.cpp

```diff
@@ -203,11 +203,13 @@ TEST(MathLib, CheckInterface_GaussAlgorithm)
 {
     boost::property_tree::ptree t_root;
     boost::property_tree::ptree t_solver;
-    t_root.put_child("LinearSolver", t_solver);
+    t_root.put_child("ogs", t_solver);
```

Comment:
Yes. When we have an internal ogs linear equations solver, this could become the tag.
For the test only "eigen", "lis", or "petsc" ptrees are passed; we are already in the "linear_solver" ptree.